src/frontend/DevolioFront/src/app/portfolio/portfolio-routing.module.ts

```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { PortfolioListComponent } from './components/portfolio-list/portfolio-list.component';
+const routes: Routes = [
+  { path: 'portfolio', component: PortfolioListComponent },
```

Comment:
path: '', right now we're routing to portfolio module at 'portfolio' and rendering portfolio list at 'portfolio' which results at 'portfiolio/portfolio'. Path here should be empty